include/openspace/documentation/documentationengine.h

```diff
@@ -88,7 +88,7 @@ class DocumentationEngine {
     static DocumentationEngine& ref();
 
     void writeJavascriptDocumentation() const;
-    void writeJsonDocumentation() const;
+    void writeJsonDocumentation(std::string folderName = "${DOCUMENTATION}") const;
```

Comment:
What's the motivation for this change? It doesn't seem to be necessary in this PR?  And using a path token as a default argument seems weird to me?

Reply:
Maybe premature optimization, but I was thinking that when we integrate with Jenkins this folder might change, so it could be useful to be able to change it. But I can remove it